src/storage/__tests__/migrations-test.js

```diff
@@ -84,10 +85,27 @@ describe('migrations', () => {
     },
   };
 
+  // What `base` becomes after migrations up through 52.
+  const base52 = {
+    migrations: { version: 52 },
+    ...base37,
```

Comment:
This will clobber `migrations`, no?

Reply:
Oops, I should switch the order of these two lines.